common/src/main/scala/com/gu/media/youtube/package.scala

```diff
@@ -56,6 +55,75 @@ package object youtube {
     }
   }
 
+  case class YouTubeMetadataUpdate(
+    title: Option[String],
+    categoryId: Option[String],
+    description: Option[String],
+    tags: List[String],
+    license: Option[String],
+    privacyStatus: Option[String]
+  ) {
+    def clean(): YouTubeMetadataUpdate = {
+      // Editorial add "- video" for on platform SEO, but it isn't needed on a YouTube video title as its a video platform
+      val cleanTitle = this.title.map(_.replaceAll(" (-|–) video$", ""))
+
+      val contentBundledTags = getContentBundlingTags()
+
+      this.copy (
+        title = cleanTitle,
+        tags = contentBundledTags
+      )
+    }
+
+    private def getContentBundlingTags(): List[String] = {
+      val contentBundlingMap: Map[String, String] = Map (
+        "uk" -> "gdnpfpnewsuk",
+        "us" -> "gdnpfpnewsus",
+        "au" -> "gdnpfpnewsau",
+        "world" -> "gdnpfpnewsworld",
+        "politics" -> "gdnpfpnewspolitics",
+        "opinion" -> "gdnpfpnewsopinion",
+        "football" -> "gdnpfpsportfootball",
+        "cricket" -> "gdnpfpsportcricket",
+        "rugby union" -> "gdnpfpsportrugbyunion",
+        "rugby league" -> "gdnpfpsportrugbyleague",
+        "f1" -> "gdnpfpsportf1",
+        "tennis" -> "gdnpfpsporttennis",
+        "golf" -> "gdnpfpsportgolf",
+        "cycling" -> "gdnpfpsportcycling",
+        "boxing" -> "gdnpfpsportboxing",
+        "racing" -> "gdnpfpsportracing",
+        "us sports" -> "gdnpfpsportus",
+        "sports" -> "gdnpfpsportother",
+        "culture" -> "gdnpfpculture",
+        "film" -> "gdnpfpculturefilm",
+        "music" -> "gdnpfpculturemusic",
+        "lifestyle" -> "gdnpfplifestyle",
+        "food" -> "gdnpfplifestylefood",
+        "health & fitness" -> "gdnpfplifestylehealthfitness",
+        "business" -> "gdnpfpbusiness",
+        "money" -> "gdnpfpmoney",
+        "fashion" -> "gdnpfpfashion",
+        "environment" -> "gdnpfpenvironment",
+        "technology" -> "gdnpfptechnology",
+        "travel" -> "gdnpfptravel",
+        "science" -> "gdnpfpscience"
+      )
+
+      val tagsAndContentBundleTags: ListBuffer[String] = new ListBuffer[String]()
```

Comment:
Could avoid the mutable list?

```scala
this.tags.flatMap { tag =>
    contentBundlingMap.get(tag.toLowerCase()) match {
        case Some(additionalTag) => List(tag, additionalTag)
        case None => List(tag)
    }
}
```

I ❤️ Scala and how much additional list allocation is required for `flatMap`